pom.xml

```diff
@@ -316,42 +392,207 @@
+   <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-oxm</artifactId>
+    <version>3.0.5.RELEASE</version>
+   </dependency>
+   <dependency>
+    <groupId>org.springframework.security</groupId>
+    <artifactId>spring-security-core</artifactId>
+    <version>${spring.security.version}</version>
+   </dependency>
+   <dependency>
+    <groupId>org.springframework.security</groupId>
+    <artifactId>spring-security-ldap</artifactId>
+    <version>${spring.security.version}</version>
+   </dependency>
+   <dependency>
+    <groupId>org.springframework.ldap</groupId>
+    <artifactId>spring-ldap-test</artifactId>
+    <version>1.3.2.RELEASE</version>
+   </dependency>
     <!-- Jetty runner dependencies -->
-    <dependency>
-      <groupId>org.mortbay.jetty</groupId>
-      <artifactId>jetty</artifactId>
-      <version>${jetty.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.mortbay.jetty</groupId>
-      <artifactId>jetty-naming</artifactId>
-      <version>${jetty.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.mortbay.jetty</groupId>
-      <artifactId>jetty-plus</artifactId>
-      <version>${jetty.version}</version>
-      <scope>test</scope>
-    </dependency>
+   <dependency>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-server</artifactId>
+    <version>${jetty.version}</version>
+   </dependency>
+   <dependency>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-jndi</artifactId>
+    <version>${jetty.version}</version>
+   </dependency>
+   <dependency>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-plus</artifactId>
+    <version>${jetty.version}</version>
+   </dependency>
+   <dependency>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-util</artifactId>
+    <version>${jetty.version}</version>
+   </dependency>
+   <dependency>
+    <groupId>tomcat</groupId>
+    <artifactId>jasper-compiler-jdt</artifactId>
+    <version>5.5.15</version>
+   </dependency>
+   <dependency>
+    <groupId>tomcat</groupId>
+    <artifactId>jasper-compiler</artifactId>
+    <version>5.5.15</version>
+   </dependency>
+   <dependency>
+    <groupId>tomcat</groupId>
+    <artifactId>jasper-runtime</artifactId>
+    <version>5.5.15</version>
+   </dependency>
   </dependencies>
   
   <properties>
-    <geotools.version>13-SNAPSHOT</geotools.version>
-    <geoserver.version>2.7-SNAPSHOT</geoserver.version>
-    <jetty.version>6.1.8</jetty.version>
-    <gsgeogig.version>2.7-SNAPSHOT</gsgeogig.version>
+    <geotools.version>15-SNAPSHOT</geotools.version>
+    <geoserver.version>2.9-SNAPSHOT</geoserver.version>
+    <spring.version>4.2.5.RELEASE</spring.version>
+    <spring.security.version>4.0.4.RELEASE</spring.security.version>
+    <jetty.version>9.2.13.v20150730</jetty.version>
+    <gsgeogig.version>2.9-SNAPSHOT</gsgeogig.version>
```

Comment:
Now that the GeoServer plugin for GeoGig is in the GeoServer community modules section, we probably don't need an explicit version for the gs-geogig artifact, and can just use the ${geoserver.version} for gs-geogig as they should be the same.